x-pack/platform/plugins/shared/onechat/public/services/data/data_service.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { HttpSetup } from '@kbn/core-http-browser';
```

Comment:
Just to be clear. We don't strictly need  `DataService` in this PR. Although it can stay. 

The other services in `/onechat/public/services` are just "client services" that wrap http api calls to server (and expose methods to call those apis) https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/onechat/public/services/tools/tools_service.ts#L28